examples/bss_example.py

```diff
@@ -168,10 +170,13 @@ def convergence_callback(Y):
         Y = pra.bss.sparseauxiva(X, S, n_iter=30, proj_back=True,
                                  callback=convergence_callback)
 
+    t_end = time.perf_counter()
+    print("Time for BSS: {:.2f}".format(t_end - t_begin))
     
     ## STFT Synthesis
     y = pra.transform.synthesis(Y, L, L, zp_front=L//2, zp_back=L//2).T
 
+
     ## Compare SDR and SIR
     sdr, sir, sar, perm = bss_eval_sources(ref[:,:y.shape[1]-L//2,0], y[:,L//2:ref.shape[1]+L//2])
     print('SDR:', sdr)
```

Comment:
We can add a separation example of the new overiva to see the performance metrics.

Reply:
Hum, I feel this will overcomplicate the script. I'll postpone this to a dedicated example...